providers/cncf/kubernetes/tests/unit/cncf/kubernetes/operators/test_pod.py

```diff
@@ -2595,17 +2595,19 @@ def test_async_get_logs_should_execute_successfully(
 
     @pytest.mark.parametrize("get_logs", [True, False])
     @patch(KUB_OP_PATH.format("post_complete_action"))
+    @patch(KUB_OP_PATH.format("client"))
     @patch(KUB_OP_PATH.format("extract_xcom"))
     @patch(HOOK_CLASS)
     @patch(KUB_OP_PATH.format("pod_manager"))
     @pytest.mark.xfail
```

Comment:
If the test is fixed by the PR, can you remove the xfail flag? Then CI should turn green as a proof
```suggestion
```